### DISCUSSION TITLE
How to add "Valve opening degree" to the card?

### DISCUSSION THREAD

**Discussion by TallDaikon** [jmcollin78/versatile-thermostat-ui-card#43] [Q&A] [answered]
I test Versatile Thermostat + this UI since today.
Other users had the Valve opening degree in the UI. How is it possible to activate it?
Here the screenshots:
https://www.kodinerds.net/thread/79320-unterst%C3%BCtzung-bei-korrekter-ansteuerung-heizk%C3%B6rperthermostat/?pageNo=2

**Comment by Laurentwb** [comment]
Unselect "Disable power infos"

**Reply by TallDaikon** [reply]
No, this shows just the Power Buttons.
![Screenshot_20250123-092455](https://github.com/user-attachments/assets/0fc275c8-f23e-467c-add7-8f662c8aed84)

**Reply by Laurentwb** [reply]
Interesting.
In my case, deselecting "Disable power infos" activates to show the opening degree


![Capture d’écran 2025-01-23 102652](https://github.com/user-attachments/assets/8dc63991-5a9b-456b-908e-d48f82703aa0)

**Reply by TallDaikon** [reply]
Thanks,
Please send me the code of your card (visual Editor) . 
It is probably
'powerEntity:'

**Comment by titomfun** [comment]
Good day,

I have the same problem. I can see the percent with the thermostat over the switch, but I don’t have the value with the SONOFF TRVZB configuration.

Did you find a solution?
Thank you

**Comment by titomfun** [comment]
Good day,
I did some tests:
The SONOFF TRVZB valve is configured via **VTherm on a climate** and not via **VTherm on a valve**.
If I configure it with VTherm on a valve, I do get a percentage indication of the valve opening.
However, when using VTherm on a climate, I don’t have any information about the valve opening percentage.

type: custom:versatile-thermostat-ui-card
entity: climate.vtherm_salon
name: Salon
disable_circle: false
disable_target_icon: false
disable_auto_fan_infos: false
set_current_as_main: false
disable_fan_only: false
disable_auto: false
disable_heat_cool: false
disable_cool: false
disable_overpowering: false
disable_window: false
disable_autoStartStop: false
disable_name: false
disable_buttons: false
disable_power_infos: false

![image](https://github.com/user-attachments/assets/23961792-dc45-4f83-a4d0-14876ed16722)

**Reply by TallDaikon** [reply, marked as answer]
Here my configuration

- type: custom:versatile-thermostat-ui-card
        entity: climate.vtgast
        disable_name: false
        disable_circle: false
        disable_autoStartStop: false
        disable_heat: false
        disable_heat_cool: false
        disable_dry: false
        disable_off: false
        disable_security_warning: false
        set_current_as_main: false
        disable_auto_fan_infos: true
        disable_buttons: false
        disable_power_infos: false
        powerEntity: number.heizgastog_valve_opening_degree
    
![IMG_20250228_103756](https://github.com/user-attachments/assets/abf0d742-0228-4b98-97ca-6d09de0460fc)

**Comment by johnbur1** [comment]
Hi,
That works but I can't see how to change the icon to a valve?

**Reply by Mark-Muc** [reply]
that will be nice to know!!
Thx
Mark-Muc
